### DISCUSSION TITLE
command 'emmet.expandAbbreviation' not found

### DISCUSSION THREAD

**Discussion by mjoffily** [coder/code-server#3642] [Q&A]
I'm getting 

command 'emmet.expandAbbreviation' not found 

when using the tab key to try to get suggestions for autocompleting HTML files. I've googled this and found some references saying it may be a bug in a version of vscode.
[https://stackoverflow.com/questions/67587366/vs-code-command-emmet-expandabbreviation-not-found](https://stackoverflow.com/questions/67587366/vs-code-command-emmet-expandabbreviation-not-found)

I'm using code-server-3.10.2-linux-amd64
Has anyone else experienced this issue?

Thanks

**Comment by jsjoeio** [comment]
Hmm...if it's a bug in VSCode you should definitely file an issue upstream. Are you able to reproduce it in VSCode?
